src/ring_theory/artinian.lean

```diff
@@ -188,12 +188,11 @@ end
 /-- A module is Artinian iff every nonempty set of submodules has a minimal submodule among them.
 -/
 theorem set_has_minimal_iff_artinian :
-  (∀ a : set $ submodule R M, a.nonempty → ∃ M' ∈ a, ∀ I ∈ a, I ≤ M' → I = M') ↔
-  is_artinian R M :=
-by rw [is_artinian_iff_well_founded, well_founded.well_founded_iff_has_min']
+  (∀ a : set $ submodule R M, a.nonempty → ∃ M' ∈ a, ∀ I ∈ a, ¬ I < M') ↔ is_artinian R M :=
+by rw [is_artinian_iff_well_founded, well_founded.well_founded_iff_has_min]
 
 theorem is_artinian.set_has_minimal [is_artinian R M] (a : set $ submodule R M) (ha : a.nonempty) :
-  ∃ M' ∈ a, ∀ I ∈ a, I ≤ M' → I = M' :=
+  ∃ M' ∈ a, ∀ I ∈ a, ¬ I < M' :=
```

Comment:
I'm not sure this is an improvement. `¬ a < b` is the correct condition in the generality of preorders, but once you get to a `partial_order` `a ≤ b → a = b` is nicer empirically.

Reply:
"A minimal element is one for which no other element is lesser". I believe my restatement of the lemma much better captures this. Besides, from `I ≤ M'` and `¬ I < M'` one can directly prove `I = M'` by using [`eq_of_le_of_not_lt`](https://leanprover-community.github.io/mathlib_docs/order/basic.html#eq_of_le_of_not_lt).

Anyways, I won't be petty about this. I thought that the current statement of this lemma reflected the statement of the lemma I deleted, and thus I changed it accordingly. But if you still think this change is unhelpful, I'll revert it.

Reply:
Yeah I think it's a matter of people not liking to write "`a` is such that if `a ≤ b` then `a = b`" on paper, even though it's better for formalisation.